apps/files_versions/lib/Command/CleanUp.php

```diff
@@ -85,46 +75,45 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
 		if (!empty($users)) {
 			foreach ($users as $user) {
-				if ($this->userManager->userExists($user)) {
-					$output->writeln("Delete versions of   <info>$user</info>");
-					$this->deleteVersions($user, $path);
-				} else {
+				if (!$this->userManager->userExists($user)) {
 					$output->writeln("<error>Unknown user $user</error>");
-					return 1;
+					return self::FAILURE;
 				}
+
+				$output->writeln("Delete versions of   <info>$user</info>");
+				$this->deleteVersions($user, $path);
 			}
-		} else {
-			$output->writeln('Delete all versions');
-			foreach ($this->userManager->getBackends() as $backend) {
-				$name = get_class($backend);
+			return self::SUCCESS;
+		}
 
-				if ($backend instanceof IUserBackend) {
-					$name = $backend->getBackendName();
-				}
+		$output->writeln('Delete all versions');
+		foreach ($this->userManager->getBackends() as $backend) {
+			$name = get_class($backend);
 
-				$output->writeln("Delete versions for users on backend <info>$name</info>");
-
-				$limit = 500;
-				$offset = 0;
-				do {
-					$users = $backend->getUsers('', $limit, $offset);
-					foreach ($users as $user) {
-						$output->writeln("   <info>$user</info>");
-						$this->deleteVersions($user);
-					}
-					$offset += $limit;
-				} while (count($users) >= $limit);
+			if ($backend instanceof IUserBackend) {
+				$name = $backend->getBackendName();
 			}
+
+			$output->writeln("Delete versions for users on backend <info>$name</info>");
+
+			$limit = 500;
+			$offset = 0;
+			do {
+				$users = $backend->getUsers('', $limit, $offset);
+				foreach ($users as $user) {
+					$output->writeln("   <info>$user</info>");
+					$this->deleteVersions($user);
+				}
+				$offset += $limit;
+			} while (count($users) >= $limit);
 		}
-		return 0;
+
+		return self::SUCCESS;
 	}
 
 
 	/**
 	 * delete versions for the given user
-	 *
-	 * @param string      $user
-	 * @param string|null $path
 	 */
 	protected function deleteVersions(string $user, string $path = null): void {
```

Comment:
```suggestion
	protected function deleteVersions(string $user, ?string $path = null): void {
```
Better to be explicit

Reply:
Sure.